patches/react-dates+17.2.0.patch

```diff
@@ -0,0 +1,4 @@
+diff --git a/node_modules/react-dates/lib/css/_datepicker.css b/node_modules/react-dates/lib/css/_datepicker.scss
```

Comment:
I was wondering if there is a better way to fix the issue with `react-dates`. Would it be possible to whitelist somewhere in the webpack config instead?

Reply:
If I'm understanding the build process correctly, the packages are built outside of webpack with a [direct call to sass.render](https://github.com/WordPress/gutenberg/blob/e215c80e1fcb613dab3b26d9a31053ae96725226/bin/packages/build-worker.js#L99), so webpack's loader config is not used here. Haven't found a way to configure this in Dart-Sass, the syntax is [determined based on file extension.](https://github.com/sass/dart-sass/blob/82b2779d5caea545d1900b2a188d462cc68dfc03/lib/src/executable/compile_stylesheet.dart#L50-L56) 

Another approach that seems to work is creating a symlink in the `date-time` component:
` _datepicker.scss` ->  `../../../../node_modules/react-dates/lib/css/_datepicker.css`

Reply:
https://github.com/WordPress/gutenberg/blob/87fa5f58534281aadd9bdf624f9b8384cfe66c89/packages/components/src/date-time/style.scss#L4

would it work if the extension was explicitly provided there?

Reply:
If it's `_datepicker.scss`, it won't find the file, if it's `_datepicker.css`, the issue remains.

Reply:
This issue still exists in the latest version:

https://unpkg.com/browse/react-dates@21.8.0/lib/css/_datepicker.css

Maybe, we should simply copy this file to the component's folder and import the local version :)